src/pages/de/core-concepts/astro-components.md

```diff
@@ -159,14 +169,23 @@ const items = ["Hund", "Katze", "Schnabeltier"];
 <ul>
   {items.map((item) => (
     <>
-      <li>Rot {item}</li>
-      <li>Blau {item}</li>
-      <li>Grün {item}</li>
+      <li>1. {item}</li>
+      <li>2. {item}</li>
+      <li>3. {item}</li>
```

Comment:
Is this change to avoid thinking about agreement between adjective and noun? I find it slightly odd to number a `<ul>`, but it doesn’t really matter 😅

Reply:
Good catch regarding the already numbered list! Fixed it using your suggestion to use the plural forms. :)